package.json

```diff
@@ -66,13 +63,12 @@
     "autoprefixer": "^9.4.5",
     "babel-jest": "^24.5.0",
     "codecov": "^3.1.0",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.11.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "es-check": "^5.0.0",
     "eslint": "^5.16.0",
```

Comment:
frontend-build should pull this in for you, you don't need an explicit dependency I think